src/igvfd/tests/data/inserts/primary_cell.json

```diff
@@ -25,9 +25,9 @@
             "GEO:SAMN45"
         ],
         "taxa": "Homo sapiens",
-        "age": "40",
+        "lower_bound_age": 40,
+        "upper_bound_age": 45,
```

Comment:
If the age was 40 then shouldn't the lower=upper=40?

Reply:
Maybe you are just trying to add diversity into the sample data?

Reply:
Yes, that was my intention.